packages/concerto-core/lib/introspect/stringvalidator.js

```diff
@@ -70,7 +71,7 @@ class StringValidator extends Validator{
                 this.regex = new CustomRegExp(validator.pattern, validator.flags);
             }
             catch(exception) {
-                this.reportError(field.getName(), exception.message);
+                this.reportError(field.getName(), exception.message,ErrorCodes.REGEX_VALIDATOR_EXCEPTION);
```

Comment:
Here I have used error codes as constants instead of inline strings for consistency